src/Stripe.net/Services/Persons/RelationshipListOptions.cs

```diff
@@ -0,0 +1,19 @@
+namespace Stripe
+{
+    using Newtonsoft.Json;
+
+    public class RelationshipListOptions : INestedOptions
+    {
+        [JsonProperty("account_opener")]
+        public bool? AccountOpener { get; set; }
```

Comment:
This property isn't present in the OpenAPI spec or supported by the API.

Reply:
It is, just not in this release but it was overlooked and since I know it will be supported it seemed cleaner to just release now.